ow_faults_injection/include/ow_faults_injection/FaultInjector.h

```diff
@@ -164,14 +111,9 @@ class FaultInjector
   bool m_arm_fault;
   bool m_ant_fault;
   bool m_cam_fault = false;
```

Comment:
These members don't seem to serve any function.  They are set, but never used.  Minor point: only the third is initialized.

Reply:
they are used in FaultInjector.cpp

Reply:
Where?

Reply:
Actually, I see only m_cam_fault used, in a meaningful sense.  The other two are just assigned but never referenced otherwise.